pipeline_dp/dp_engine.py

```diff
@@ -411,16 +412,21 @@ def unnest_cross_partition_bound_sampled_per_key(pid_pk_v):
         return self._backend.flat_map(
             col, unnest_cross_partition_bound_sampled_per_key, "Unnest")
 
-    def _select_private_partitions_internal(self, col,
-                                            max_partitions_contributed: int,
-                                            max_rows_per_privacy_id: int):
+    def _select_private_partitions_internal(
+        self,
+        col,
+        max_partitions_contributed: int,
+        max_rows_per_privacy_id: int,
+        partition_selection_method: str = pipeline_dp.PartitionSelectionMethod.
```

Comment:
Nit: remove default argument, anyway this function is called from `aggregate()` with all arguments set